### ISSUE TITLE
Kweb packaging issues

### ISSUE THREAD

**Issue by alkoclick** [kwebio/kweb-core#117]
- Kweb packages its `logback.xml` file in the jar, this messes up with everyone's logback configs. This is a discouraged practice. The opinionated defaults, especially setting the root logger to error, can cause loss of information.

```
13:44:53,188 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Could NOT find resource [logback-test.xml]
13:44:53,188 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Could NOT find resource [logback.groovy]
13:44:53,189 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Found resource [logback.xml] at [jar:file:/app/libs/core-0.6.4.jar!/logback.xml]
13:44:53,189 |-WARN in ch.qos.logback.classic.LoggerContext[default] - Resource [logback.xml] occurs multiple times on the classpath.
13:44:53,189 |-WARN in ch.qos.logback.classic.LoggerContext[default] - Resource [logback.xml] occurs at [jar:file:/app/libs/REDACTED.logger-0.12.12.jar!/logback.xml]
13:44:53,189 |-WARN in ch.qos.logback.classic.LoggerContext[default] - Resource [logback.xml] occurs at [jar:file:/app/libs/core-0.6.4.jar!/logback.xml]
```
Suggested action: Just delete `logback.xml` from src/main/resources.

- The Kweb artifact is named "core", which produces unintuitive error messages, such as the above - This issue is a regression of #62. Not sure what broke it, but we should probably be setting an artifact name somewhere to avoid this in the future?

**Comment by sanity** [comment]
I just released `0.7.6` with the logback files removed but I'm not sure what you're referring to with regard to the artifact name being `core` - the repository is called "kweb-core" as is the `rootProject.name` in `settings.gradle`.

Because the repository was renamed from `core` to `kweb-core`, Github will still redirect from `https://github.com/kwebio/core` to the current repository name - perhaps the old repo name is still being used somewhere?

**Comment by alkoclick** [comment]
You are correct, the artifact thing was on my end. I had 

```
implementation "com.github.kwebio:core:$kwebV"
```

instead of `implementation "com.github.kwebio:kweb-core:$kwebV"` in my build.gradle. Thanks!
